.github/workflows/ci.yml

```diff
@@ -183,7 +183,16 @@ jobs:
         run: cargo build
       - name: Check diff
         run: |
-          [[ -z $(git status --porcelain loadgen/kitchensink/ workers/python/protos/) ]] || (git diff; echo "Protos changed"; false) 1>&2
+          git config --global core.safecrlf false
+          git diff > generator.diff
+          git diff --exit-code
+      - name: Upload generator diff
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: generator-diff
+          path: generator.diff
+          if-no-files-found: ignore
```

Comment:
This was a suggestion I added to get the protoc results from the CI, download them and apply them locally. Managing to get the right version locally, at least on OSX, is a nightmare.

Reply:
Could add it to the README somewhere